frontend/src/component/FeedItem/FeedItemFooter.jsx

```diff
@@ -22,24 +24,32 @@ export class FeedItemFooter extends React.Component {
     return (
       <div>
         <div className='feedItemFooter__right'>
-          {props.commentList.length}
-          <IconButton
-            icon='far fa-comment'
-            text={props.t('Comment')}
-            intent='link'
-            onClick={this.handleCommentClicked.bind(this)}
-            dataCy='feedItemFooter__comment'
+          <EmojiReactions
+            apiUrl={FETCH_CONFIG.apiUrl}
+            loggedUserId={props.user.userId}
+            contentId={props.content.content_id}
+            workspaceId={props.content.workspace_id}
           />
+          <div className='feedItemFooter__comments'>
+            {props.commentList.length}
+            <IconButton
+              icon='far fa-comment'
+              text={props.t('Comment')}
+              intent='link'
+              onClick={this.handleCommentClicked.bind(this)}
```

Comment:
do you need to bind this here?
Can you try without? I see that you didn't add it

Reply:
This is a chunk of code I moved, should I change this anyway?